src/test/java/CRTGearRatioUtilTestingStuff.java

```diff
@@ -0,0 +1,43 @@
+import frc.robot.utils.CRTGearRatioUtil;
+import org.junit.jupiter.api.Test;
+import java.util.Optional;
+
+public class CRTGearRatioUtilTestingStuff {
+  @Test
+  void test() {
+    assert checker(44, 42, 22);
+    assert checker(44, 26, 22);
+    assert !checker(3, 4, 22);
+    assert !checker(4,3,10);
+    assert checker(28,30,14);
```

Comment:
Can you leave a comment on some of these stating why we should believe that it should work or should not work, whatever we're asserting?

Reply:
done